vars/Fedora.yml

```diff
@@ -0,0 +1,6 @@
+---
+# vars file for RedHat
```

Comment:
This comment does not match the filename.

Reply:
See commit b2f46de.